js/core/src/reflection.ts

```diff
@@ -226,10 +226,9 @@ export class ReflectionServer {
 
     server.post('/api/notify', async (request, response) => {
       const { telemetryServerUrl, reflectionApiSpecVersion } = request.body;
-      if (typeof telemetryServerUrl === 'string') {
-        setTelemetryServerUrl(telemetryServerUrl);
-        logger.debug(`Connected to telemetry server on ${telemetryServerUrl}`);
-      }
+      const url = process.env.GENKIT_TELEMETRY_SERVER ?? telemetryServerUrl;
```

Comment:
Can you please also add

```ts
if (process.env.GENKIT_TELEMETRY_SERVER) {
  setTelemetryServer(process.env.GENKIT_TELEMETRY_SERVER)
}
```
to https://github.com/firebase/genkit/blob/7cb0116598b338c4e43a8c3136fadc4cf48a39da/js/core/src/tracing.ts#L51